### ISSUE TITLE
Compare configs unicode encode error

### ISSUE THREAD

**Issue by g7gpr** [CroatianMeteorNetwork/RMS#372]
Observed in logs on a Pi5 running standard distribution on prerelease branch.

```
2024/08/20 22:27:36-DEBUG-Reprocess-line:464 - ['Traceback (most recent call last):\n', 
'  File "/home/rms/source/RMS/RMS/Reprocess.py", line 456, in processNight\n    
                   f.write(compareConfigs(config_file_path,\n', "UnicodeEncodeError: 'latin-1' codec can't encode character '\\u2022' in position 570: ordinal not in range(256)\n"]
```

Results in an empty file

```
-rw-r--r-- 1 rms    0 Aug 20 22:27 AU0004_20240820_101725_619305_config_audit_report.txt
```

**Comment by Cybis320** [comment]
Is this solved with the #360 merge?

**Comment by g7gpr** [comment]
Last seen on 20240822, so yes.
